index.js

```diff
@@ -11,7 +11,9 @@ function reducePropsToState(propsList) {
 }
 
 function handleStateChangeOnClient(title) {
-  document.title = title || '';
+  if (title !== document.title) {
```

Comment:
Let’s calculate `nextTitle = title || ''` beforehand so that this check is more consistent (right now we compare and set potentially different values).